guby_backend/templates/app/meetups.html

```diff
@@ -28,45 +28,47 @@
-                    <div class="card-body">
-                        <div class="row">
-                            <div class="col-sm-9">
-                                <h4 class="card-title">Your Meetups</h4>
-                            </div>
-                            <div class="col-sm-3">
-                                <a class="btn btn-primary" href="/app/meetups/add">Add</a>
-                            </div>
-                        </div>
-                        <table class="table ">
-                            <thead>
-                              <tr>
-                                <th scope="col">Meetup Name</th>
-                                <th scope="col">Action</th>
-                              </tr>
-                            </thead>
-                            <tbody>
-                              {% for meetup in meetups %}
-                                <tr>
-                                    <td><a href="/app/meetups/{{ meetup.id }}/events">{{ meetup.name }}</a></td>
-                                    <td>
-                                        <a href="/app/meetups/{{ meetup.id }}/edit">Edit</a>
-                                        <a href="/app/meetups/{{ meetup.id }}/delete">Delete</a>
-                                        <a href="/app/meetups/{{ meetup.id }}/owner/">Manage Owners</a>
-                                    </td>
-                                </tr>
-                              {% endfor %}
-                            </tbody>
-                          </table>
+      <div class="row">
+        <div class="col-6 mx-auto">
+          <div class="card">
+            <div class="card-body">
+              <h1>Meetups</h1>
+              <div class="card mt-3">
+                <a href="/app/meetups/add">
+                <div class="card-body d-flex justify-content-center">
+                  <i data-feather="plus"></i>
+                  &nbsp;Create your meetup
+                </div>
+                </a>
+              </div>
+              {% for meetup in meetups %}
+              <div class="card mt-3">
+                <div class="card-body">
+                  <h3 class="card-title">{{ meetup.name }}</h3>
+                  <a class="btn btn-primary btn-sm" href="/app/meetups/{{ meetup.id }}/events">Manage events</a>
+                  <div class="btn-group">
+                    <a class="btn btn-sm btn-secondary" href="/app/meetups/{{ meetup.id }}/edit">Edit</a>
+                    <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      <span class="sr-only">Toggle Dropdown</span>
+                    </button>
+                    <div class="dropdown-menu">
+                      <a class="dropdown-item" href="/app/meetups/{{ meetup.id }}/delete">Delete</a>
```

Comment:
You need to add link to meetup edit page

Reply:
That's there [here](https://github.com/unnati-xyz/guby/pull/9/files/f5f543465a30d5d827fe86a5680b6737541e45bc#diff-213d8ce67c316255b1c07c27befb4348R54) 

Its a split dropdown with the primary action being 'edit' and 'delete' and 'manage owners' are secondary options

![image](https://user-images.githubusercontent.com/240368/87240993-34bb6680-c43c-11ea-839a-ba0601ac23d1.png)

Reply:
Oh right! To me it looked like I always need to select from dropdown :)

Reply:
It is confusing, yes. Let us ship this and check back if folks are confis